src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c

```diff
@@ -682,12 +682,14 @@ static void xocl_dev_percpu_kill(void *data)
 
 	percpu_ref_kill(ref);
 }
-
+#if KERNEL_VERSION(5, 3, 0) > LINUX_VERSION_CODE && \
```

Comment:
Why do we need this change?

Reply:
Compiler complains about xocl_dev_percpu_kill is unused when 
4.20.2>LINUX_KERNEL_VERSION > 4.15.0

because of

#if KERNEL_VERSION(5, 3, 0) > LINUX_VERSION_CODE && \
	(LINUX_VERSION_CODE >= KERNEL_VERSION(4, 20, 2))
	xdev->pgmap.kill = xocl_dev_pgmap_kill_nop;
#endif